net/scripts/gce-provider.sh

```diff
@@ -208,10 +208,11 @@ cloud_DeleteInstances() {
   declare names=("${instances[@]/:*/}")
   declare zones=("${instances[@]/*:/}")
 
-  (
+  for zone in "${zones[@]}"; do
     set -x
-    gcloud beta compute instances delete --zone "${zones[0]}" --quiet "${names[@]}"
-  )
+    # Try deleting instances in all zones
+    gcloud beta compute instances delete --zone "$zone" --quiet "${names[@]}" || true
```

Comment:
What is the purpose of the "|| true"?

Reply:
Ignore the error of the command. If it fails to delete some nodes in one zone, it'll continue with deleting in other zones.

Reply:
Do we want to act on that failure, or just suppress and move on?  One teardown failure should not kill the script, but does that leave us with a dangling node?

Reply:
I think failures in deletion in one zone should not prevent the script from deleting nodes in other zones. Some sort of audit towards the end and reporting will be useful (i.e. if some node is still lingering around, notify the user).